r/src/main/java/org/molgenis/r/RServerConnectionFactory.java

```diff
@@ -15,17 +16,45 @@ public RServerConnectionFactory(EnvironmentConfigProps environment) {
     this.environment = environment;
   }
 
-  @Override
-  public RServerConnection tryCreateConnection() {
+  int doHead(String uri) {
```

Comment:
why change 'public' to be friendly ?

Reply:
Not sure I understand ... should `int doHead()` be public?

Reply:
I see now ... it was already public ... guess this is resolved?